README.rst

```diff
@@ -1,39 +1,40 @@
-The Chrome Extension for codesy.io
-==================================
-The codesy.io Chrome Extension adds codesy.io information to open-source bugs
+The codesy.io widget
+====================
+The codesy.io widget is Chrome Extension and Firefox Addon the adds codesy.io information to open-source bugs
 you visit.
 
 
-To use the extension 'unpacked'
--------------------------------
-1. Clone this repo
-2. Copy the prod/manifest.json into the root of the project
-   (You can also run gulp dev-start)   
-3. Follow these instructions (http://developer.chrome.com/extensions/getstarted.html#unpacked)
-
-
 Development
 -----------
 1. Clone this repo
 2. Set up Gulp
-
    * node install gulp
    * 'npm install' to install dependencies
+3. For firefox run 'npm install jpm'
```

Comment:
We may as well add `jpm` to the `package.json` so a single `npm install` will install everything. Installing this way is only for devs anyway.